packages/scandipwa/src/component/ProductActions/ProductActions.style.scss

```diff
@@ -48,6 +48,7 @@
 
     &-Title {
         margin-block: 8px;
+        text-transform: none;
```

Comment:
Isn't this property redundant, since nothing is transformed by default?

Reply:
text-transform: capitalize is applied universally to all h1 elements in reset.scss

Reply:
Got it! Thanks!